dashboard/app/views/layouts/_header.html.haml

```diff
@@ -25,6 +25,18 @@
               = render partial: 'shared/header_progress'
             - elsif @edit_blocks
               .header_level{style:'font-size: 250%; line-height: 125%;'} Editing #{@edit_blocks.humanize}
-          - if params[:controller] !~ /^sessions/
-            .header_user
-              = render partial: 'shared/user_header'
+          .header_right
+            %span.stage_links{style: 'display: none;'}
```

Comment:
This change doesn't change when the free play links show up.  It looks like it's defined as `script.twenty_hour? || script.stages.to_a.count > 1`.